test/system/760-system-renumber.bats

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# tests for podman system renumber
+#
+
+load helpers
+
+@test "podman system renumber - Basic test with a volume" {
+    run_podman volume create test
+    run_podman system renumber
+    run_podman volume rm test
```

Comment:
```suggestion
    run_podman volume create test
    assert "$output" == "test" "podman volume create output"
    run_podman system renumber
    assert "$output" == "" "podman system renumber output"
    run_podman volume rm test
    assert "$output" == "test" "podman volume rm output"
```

Please add some output checks as well to make sure we don't print anything unexpected.